specification/netapp/resource-manager/Microsoft.NetApp/stable/2020-06-01/netapp.json

```diff
@@ -1459,255 +1618,2049 @@
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "#/parameters/AccountName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/backupPoliciesList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-examples": {
+          "Backups_List": {
+            "$ref": "examples/BackupPolicies_List.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NetApp/netAppAccounts/{accountName}/backupPolicies/{backupPolicyName}": {
+      "get": {
+        "tags": [
+          "Backup Policy"
+        ],
+        "summary": "Get a backup Policy",
+        "description": "Get a particular backup Policy",
+        "operationId": "backupPolicies_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "#/parameters/AccountName"
+          },
+          {
+            "$ref": "#/parameters/BackupPolicyName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/backupPolicyDetails"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-examples": {
+          "Backups_Get": {
+            "$ref": "examples/BackupPolicies_Get.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Backup Policy"
+        ],
+        "summary": "Create a backup policy",
+        "description": "Create a backup policy for Netapp Account",
+        "operationId": "backupPolicies_Create",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "#/parameters/AccountName"
+          },
+          {
+            "$ref": "#/parameters/BackupPolicyName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "description": "Backup policy object supplied in the body of the operation.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/backupPolicyPatch"
```

Comment:
for a PUT http method, the request and reponse model must be same. if there are properties which cant be set in the request, they can be marked as readonly.

Reply:
Fixed